mne/io/nirx/nirx.py

```diff
@@ -19,15 +19,28 @@
 
 
 @fill_doc
-def read_raw_nirx(fname, preload=False, verbose=None):
+def read_raw_nirx(fname, saturated='ignore', preload=False, verbose=None):
     """Reader for a NIRX fNIRS recording.
 
-    This function has only been tested with NIRScout devices.
+    This function has only been tested with NIRScout and NIRSport1 devices.
```

Comment:
I'd move this in a separate Notes section (see e.g. https://github.com/mne-tools/mne-python/blob/master/mne/io/edf/edf.py#L70).

Reply:
Right. Shall I also move the explanation regarding the saturation in the Notes section?

Reply:
Yes please.